docs/setup/pip.md

```diff
@@ -16,7 +16,7 @@ We support :simple-python: **Python from 3.8 to 3.12**.
     wget -O "$TEMP_DEB" "https://github.com/KhiopsML/khiops/releases/download/10.2.0/khiops-core_10.2.0-1-${CODENAME}.amd64.deb" && \
```

Comment:
Perhaps add somewhere a paragraph that a supported Python (>= 3.8) needs to be installed first, for Rocky 8 (vanilla Python on this OS is 3.6).

Reply:
Done

Reply:
Done.